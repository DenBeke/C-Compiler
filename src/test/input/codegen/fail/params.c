// Function parameters not matching

int a(int x, int y) {
    return x + y;
}

void main() {
    
    
    a(5);
    
}