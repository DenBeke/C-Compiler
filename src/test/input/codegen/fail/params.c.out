[31m[ERROR] line 10: Number of arguments for 'a': 2, 1 given[0m
